### DISCUSSION TITLE
GRAPESJS incrementing the Id

### DISCUSSION THREAD

**Discussion by lifeofsa** [GrapesJS/grapesjs#5440] [Q&A]
GRAPESJS builder incrementing the Id of a form although both forms are on different pages, I want to send a form with the same Id for both pages and I dont know what to do.

Here are the ScreenShots.

On the checkout page I'm sending the form id "billing-form".

![checkout](https://github.com/GrapesJS/grapesjs/assets/83239811/77c29a18-ed8a-49ac-9f2d-cdd75241ccc4)

But when I tried to send the same id on upsale page it just changes the id from "billing-form" to "billing-form-2". and I don't want 
that I want to send the same id for both pages Kindly someone help.?

![Upsale](https://github.com/GrapesJS/grapesjs/assets/83239811/0f95013b-225a-4eec-8f65-6e226a1d6e6c)

**Comment by lifeofsa** [comment]
@artf ? kindly help me

**Reply by nguyenvoanhkhoa1** [reply]
id is unique, I think you should use class selector instead
